src/app.ts

```diff
@@ -49,7 +49,12 @@ const App = (config: AppConfig,
   const app = express();
   app.set('port', process.env.PORT || 3000);
   app.use(bodyParser.json());
-  app.use(fileUpload());
+  app.use(
+    fileUpload({
```

Comment:
throw error as far up as possible. 
using the busboy option style for `express-fileupload`